.github/workflows/main.yml

```diff
@@ -104,7 +104,12 @@ jobs:
        with:
          go-version: '^1.17'
      - name: generate mocks
-       run: |
+       uses: nick-invision/retry@7f8f3d9f0f62fe5925341be21c2e8314fd4f7c7c
+       with:
+          max_attempts: 3
+          retry_on: error
+          timeout_minutes: 30
+          command: |
```

Comment:
Why do we need multiple retry? Shouldn't this only be needed for `build-proto`?

Reply:
These individual jobs have been failing, and that is the reason.